### PR TITLE
No Keithley for the experiment with field scan

### PR DESCRIPTION
**Pull request by debobiswas01**
changed timing for dewar to 60*60
removed Keithley

### CODE REVIEW

File: experiment_without_Keithley.py [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    # During a field ramp from 50,000 to 0 Oe at 20 Oe/sec
+    Scan_Field(0)
+    
+    print("waiting for dewar pressure (60 mins)")
+    time.sleep(60 * 60)  # wait for dewar pressure to stabilize
+    
+    # Polling temperature/field and performing resistivity measurement 
+    # During a field ramp from 0 to -50,000 Oe at 20 Oe/sec
+    Scan_Field(-50000)
+
```
Comment (comp-phys-marc) [inline experiment_without_Keithley.py]: Should we wait for the dewar pressure here as well?

File: experiment_without_Keithley.py [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+        0.0,
+        20,
+        client.field.approach_mode.linear,
+        client.field.driven_mode.driven
+    )
+    client.set_temperature(
+        300,
+        2,
+        client.temperature.approach_mode.fast_settle
+    )
```
Comment (comp-phys-marc) [inline experiment_without_Keithley.py:159]: Please use wait_for here as well.

**Review by comp-phys-marc — CHANGES_REQUESTED** [review]
A few comments on timing.

File: experiment_without_Keithley.py (hunk 1 above)
Comment (debobiswas01) [reply experiment_without_Keithley.py]:
```suggestion
    #waiting 60 mins for dewar pressure
    time.sleep(60*60)
```

File: experiment_without_Keithley.py (hunk 2 above)
Comment (debobiswas01) [reply experiment_without_Keithley.py:159]:
```suggestion
    )
    print('Waiting...')
    client.wait_for(
        60,
        timeout_sec=0,
        bitmask=client.temperature.waitfor | client.field.waitfor
    )
```

**Review by comp-phys-marc — APPROVED** [review]

**Pull request merged**